assets/infrastructure-providers/infrastructure-vsphere-provider.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: operator.cluster.x-k8s.io/v1alpha2
+kind: InfrastructureProvider
+metadata:
+  creationTimestamp: null
+  name: vsphere
+  namespace: openshift-cluster-api
+spec:
+  deployment:
+    containers:
+    - command:
+      - ./bin/cluster-api-provider-vsphere-controller-manager
+      name: manager
+  fetchConfig:
+    selector:
+      matchLabels:
+        provider.cluster.x-k8s.io/name: vsphere
+        provider.cluster.x-k8s.io/type: infrastructure
+  version: v1.7.0
```

Comment:
We now have `v1.8.3` in payload. I'd be better to switch to that for compatibility.